addons/account_edi/models/account_move.py

```diff
@@ -127,10 +127,23 @@ def _compute_edi_show_abandon_cancel_button(self):
         for move in self:
             move.edi_show_abandon_cancel_button = any(doc.edi_format_id._needs_web_services()
                                                       and doc.state == 'to_cancel'
-                                                      and move.is_invoice(include_receipts=True)
+                                                      and move._is_edi_invoice(include_receipts=True)
                                                       and doc.edi_format_id._is_required_for_invoice(move)
                                                       for doc in move.edi_document_ids)
 
+    def _is_edi_invoice(self, include_receipts=False):
+        """To be overridden for special cases where entry types need to be sent. """
+        self.ensure_one()
+        return self._get_edi_document_type(include_receipts=include_receipts) == 'invoice'
+
+    def _get_edi_document_type(self, edi_format=None, include_receipts=False):
```

Comment:
I would not pass the include_receipts param @odoostan

Reply:
Done.
Hard to say what modules may be affected, if any, since this changes the default behavior.

Reply:
I did a quick test just in case, and it seems to be working OK.
I did the same tests as before.
@odoostan @jco-odoo